### PR TITLE
Add `neptune_scale.projects.create_project()`

### PR DESCRIPTION
**Pull request by kgodlewski**

### CODE REVIEW

File: src/neptune_scale/exceptions.py [hunk 1]
```diff
@@ -121,7 +141,7 @@ class NeptuneInvalidCredentialsError(NeptuneScaleError):
     In the terminal:
         {bash}export NEPTUNE_API_TOKEN="YOUR_API_TOKEN"{end}
 
-    {correct}Windows{end}
+    {correct}WindowGs{end}
```
Comment (PatrykGala) [inline src/neptune_scale/exceptions.py:144]: ?

**Review by PatrykGala — CHANGES_REQUESTED** [review]
WindowsGs?

File: src/neptune_scale/exceptions.py (hunk 1 above)
Comment (kgodlewski) [reply src/neptune_scale/exceptions.py]: Cat on my keyboard?🤦 Fixed.

**Review by PatrykGala — APPROVED** [review]

**Review by PatrykGala — APPROVED** [review]
lgtm

File: src/neptune_scale/projects.py [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+            The default is "priv".
+        description: Project description.
+            If None, it will be left empty.
+        key: Project identifier. Must contain 1-10 upper case letters or numbers (at least one letter).
+            For example, "CLS2". If you leave it out, Neptune generates a project key for you.
+        fail_if_exists: If the project already exists and this flag is True, an error is raised.
+        api_token: Account's API token.
+            If None, the value of the NEPTUNE_API_TOKEN environment variable is used.
+            Note: To keep your token secure, use the NEPTUNE_API_TOKEN environment variable rather than placing your
+            API token in plain text in your source code.
```
Comment (normandy7) [inline src/neptune_scale/projects.py:47]:
```suggestion
        name (str): Name of the project. Can contain letters and hyphens (-). For example, "project-x".
        workspace (str, optional): Name of your Neptune workspace.
            You can omit this argument if you include the workspace name in the `name` argument.
        visibility: Level of privacy for the project. Options:
            - "pub": Public. Anyone on the internet can see it.
            - "priv" (default): Private. Only users specifically assigned to the project can access it. Requires a plan with
                project-level access control.
            - "workspace" (team workspaces only): Accessible to all workspace members.
        description: Project description. If None, it's left empty.
        key: Project identifier. Must contain 1-10 upper case letters or numbers (at least one letter).
            For example, "PX2". If you leave it out, Neptune generates a project key for you.
        fail_if_exists: If the project already exists and this flag is set to `True`, an error is raised.
        api_token: Account's API token.
            If not provided, the value of the NEPTUNE_API_TOKEN environment variable is used (recommended).
```

**Review by normandy7 — APPROVED** [review]

**Pull request merged**